examples/networking/misc/lwip_ipv4/main.c

```diff
@@ -0,0 +1,171 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Krzysztof Cabaj <kcabaj@gmail.com>
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       lwip_ipv4 - sample application for demonstrating basic LWIP
+ *              IPv4 client/server functions.
+ *
+ * @author      Krzysztof Cabaj <kcabaj@gmail.com>
+ *
+ * @}
+ */
+#include "stdio.h"
+#include "stdlib.h"
+#include "shell.h"
+#include "net/sock/udp.h"
+#include <arpa/inet.h>
+#include "thread.h"
+#include "mutex.h"
+
+#define CLIENT_TIMEOUT  3 /* in seconds */
```

Comment:
```suggestion
#define CLIENT_TIMEOUT_SEC  3 /* in seconds */
```

so the unit is obvious